metrics/density/README.md

```diff
@@ -7,7 +7,10 @@ This number of containers could be working under different configurations/condit
 - `docker_memory_usage.sh`: measures the Proportional Set Size (PSS) memory average
    for a number of containers launched in idle mode. This test uses the `sleep` command
    to allow any memory optimizations to 'settle' (e.g. KSM execution) for a configurable
-   period of time.
+   period of time. It also has an optional 'auto' mode which will try to determine when KSM
+   has settled down by examining the KSM `full_scans` and `pages_shared` values. auto mode
```

Comment:
Lines 11-13 suggested rewrite: Auto mode detection of KSM settling overrides the timeout value to take the measurements and to complete the test.

Reply:
@klynnrif I've re-written that section - hopefully that is now clearer?

Reply:
@grahamwhaley it is - I had a couple suggestions for the new section to make the voice active. Thanks!